api/package.json

```diff
@@ -16,6 +16,7 @@
   },
   "dependencies": {
     "async": "^3.2.0",
+    "axios": "^0.19.2",
```

Comment:
Please use `request-promise-native` that's already installed. We have not decided on which replacement to use. https://github.com/medic/cht-core/issues/6250